java/openjfx11/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            openjfx11
+version         11.0.2
+revision        0
+
+set build       0
+set major       11
+
+categories      java devel
+maintainers     {@lhaeger} openmaintainer
+platforms       darwin
+license         GPL-2
+supported_archs x86_64
+
+description     OpenJFX ${major}
+
+long_description OpenJFX is an open source, next generation client application platform \
+                 for desktop, mobile and embedded systems built on Java.
+
+homepage        https://openjfx.io/
+
+depends_lib     port:openjdk11
+
+master_sites    https://download2.gluonhq.com/openjfx/${version}/
+distname        openjfx-${version}_osx-x64_bin-sdk
+use_zip         yes
+
+checksums       rmd160  3b74b4bd563cc46e28b886df5c9e239fb00845c2 \
+                sha256  e98158812db1a0037cdaf85824adff384e41e3edf046fda145479ce6057cb514 \
+                size    41684332
+
+worksrcdir      javafx-sdk-${version}
+
+use_configure    no
+
+build {}
+
+# macOS Java tools expect to find Java virtual machines under /Library/Java/JavaVirtualMachines, which is not under ${prefix}.
+destroot.violate_mtree yes
+
+set target /Library/Java/JavaVirtualMachines/openjdk11/Contents/Home
+set destroot_target ${destroot}${target}
```

Comment:
Can you install it into `${prefix}/Library/Java/JavaVirtualMachines/openjdk11/Contents/Home` and symlink to the `/Library/..` directory? I believe other ports work that way but I don't have an example handy.

Reply:
Nevermind. I see that at least one other similar port (`graalvm`) does it this way.

Reply:
I took openjdk8 as a template, which does it the same way. Since openjfx is basically a "plugin" to openjdk, I'd prefer to keep it as similar as possible.